rskj-core/src/test/java/co/rsk/peg/performance/RegisterBtcTransactionTest.java

```diff
@@ -110,8 +110,8 @@ private BridgeStorageProviderInitializer generateInitializerForLock(int minBtcBl
             // Sender and amounts
             BtcECKey from = new BtcECKey();
             Address fromAddress = from.toAddress(networkParameters);
-            Coin fromAmount = Coin.CENT.multiply(Helper.randomInRange(10, 100));
-            Coin lockAmount = fromAmount.divide(Helper.randomInRange(2, 10));
+            Coin fromAmount = Coin.CENT.multiply(Helper.randomInRange(100, 150));
+            Coin lockAmount = fromAmount.divide(Helper.randomInRange(2, 4));
```

Comment:
Is this change required to fix the test?

Reply:
Yes, with the old values, the test didn't work.